### PR TITLE
Update contibuting.md: contribution shall not impact code coverage

### PR DESCRIPTION
**Pull request by JeanRochCoulon**
One addional condition has been added to contributing.md. A PR can be merged only if it does not impact the code coverage to guaranty the RTL maturity level.

### CODE REVIEW

**Review by jquevremont — CHANGES_REQUESTED** [review]
Excellent PR. But I did not clearly understand the sentence.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Therefore here are guidelines to help the CVA6 team accept new contributions:
         - We call this the "parachute" rule: The CVA6 team does not want to use it but is far more comfortable getting one.
     * Your contribution shall pass the Continuous Integration (CI) flow
         - When the contribution is disabled: in all cases, to ensure you have not broken the design.
+        - When the contribution is disabled: the previous line and condition code coverage shall not be impacted.
```
Comment (jquevremont) [inline CONTRIBUTING.md:39]: I do not understand "the previous line".

Comment (JeanRochCoulon) [reply CONTRIBUTING.md:39]: "previous" qualifies "coverage". But as it is not clear, I will remove the word 'previous'.

**Review by jquevremont — APPROVED** [review]
LGTM

**Pull request merged**